extra-libs/openexr/spec

```diff
@@ -1,3 +1,3 @@
-VER=2.5.2
+VER=2.5.4
 SRCTBL="https://github.com/AcademySoftwareFoundation/openexr/archive/v${VER}.tar.gz"
-CHKSUM="sha256::5da8dff448d0c4a529e52c97daf238a461d01cd233944f75095668d6d7528761"
+CHKSUM="sha256::dba19e9c6720c6f64fbc8b9d1867eaa75da6438109b941eefdc75ed141b6576d"
```

Comment:
Use new SRCS and CHKSUMS format.

Reply:
I would prefer not switching over if all this need is only a single tarball. SRCS doesn't really improve readability in this case.